abis/Steward.json

```diff
@@ -1094,7 +1094,7 @@
   {
     "constant": true,
     "inputs": [],
-    "name": "globalBenefactorPeriodicWithdrawalLimit",
+    "name": "globalbeneficiaryPeriodicWithdrawalLimit",
```

Comment:
Damn, I'll need to fix this. I bet there's a way to search for instances of 'beneficiary' that have letters in front of them and replace